packages/core/src/anchor/ethereum/__tests__/ethereum-anchor-service.retry-failure.test.ts

```diff
@@ -38,14 +38,14 @@ test('re-request an anchor till get a response', async () => {
   const loggerProvider = new common.LoggerProvider()
   const diagnosticsLogger = loggerProvider.getDiagnosticsLogger()
   const errSpy = jest.spyOn(diagnosticsLogger, 'err')
-  const anchorService = new eas.EthereumAnchorService(
-    'http://example.com',
-    diagnosticsLogger,
-    100
-  )
+  const anchorService = new eas.EthereumAnchorService('http://example.com', diagnosticsLogger, 100)
   let lastResponse: any
   const subscription = anchorService
-    .requestAnchor(FAKE_STREAM_ID, FAKE_CID)
+    .requestAnchor({
+      streamID: FAKE_STREAM_ID,
+      tip: FAKE_CID,
+      timestamp: Date.now(),
```

Comment:
Looks like it's ms. We use seconds in other places in the code base so either we conform to that or we make it really clear for anyone consuming this param that it will be in ms.

Reply:
Maybe consider renaming the param to `timestampMs`?